contracts/src/utils/context.rs

```diff
@@ -0,0 +1,17 @@
+//! Context functions used for mocking unit tests.
+
+use stylus_sdk::{alloy_primitives::Address, msg};
+
+/// Returns the address of the message sender.
+#[cfg(test)]
+pub fn msg_sender() -> Address {
+    motsu::prelude::Context::current()
+        .get_msg_sender()
+        .expect("msg_sender should be set")
```

Comment:
Devs would then need to copy/paste these `msg_sender` definitions in order to take advantage of motsu's mocking. 
Since these are so tightly related to motsu, would it make sense to make these helpers _a part_ of motsu?

motsu would then be a testing library that provides wrappers for `msg::sender`, `msg::value`, `block::timestamp` etc. that devs can use to be able to then mock these in their motsu (unit) tests.

Pros: devs can just use these helpers; easier to setup and use motsu; clearer documentation.
Cons: once the Stylus team adds ability to mock these from the SDK, it will make our wrappers redundant - devs using motsu will have a breaking change at some point.

 @bidzyyys @qalisander wdyt?

NOTE: given that the Stylus team will enable mocking these values, thus making our wrappers redundant, it makes our efforts to mock these temporary, whether we make these helpers part of motsu or not.

Reply:
>would it make sense to make these helpers a part of motsu?
Motsu is a part of `[dev-dependencies]`. In this case we need make motsu available for non-dev `[depencencies]`, or introduce the other helper module.
p.s. `msg_sender()` abstraction should be available as a non development dependency. It is not just for tests.

> NOTE: given that the Stylus team will enable mocking these values, thus making our wrappers redundant, it makes our efforts to mock these temporary, whether we make these helpers part of motsu or not.
Correct @0xNeshi, this was a temporary solution, since I didn't know how happy OL team about stylus sdk cache reset. We will remove it once stylus sdk be ready.

Reply:
Yeah, it would make sense to make these helpers a non-dev dependency then. 

We can keep things as-is for now, as it's possible OL will update their SDK to allow mocking these values soon.
We shouldn't invest too much time into this imo.